api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
What is this change and why do we need it?

Reply:
- it seems recent versions of go tools are prefixing `go:` for code generation 
- this shouldn't cause any issue for us

Reply:
ok